sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -396,6 +406,11 @@ public int maxRequestsPerChannel() {
             return this.maxRequestsPerChannel;
         }
 
+        public int maxConcurrentRequestsPerEndpoint()
+        {
```

Comment:
code style `{` on the same line as method name

Reply:
Fixed